ci/prow/periodic_config.go

```diff
@@ -246,6 +246,7 @@ func generateVersionBumpertoolPeriodicJob() {
 	addExtraEnvVarsToJob(&data.Base)
 	configureServiceAccountForJob(&data.Base)
 	addVolumeToJob(&data.Base, "/etc/prow-auto-bumper-github-token", "prow-auto-bumper-github-token", true)
+	addVolumeToJob(&data.Base, "/root/.ssh", "prow-updater-robot-ssh-key", true)
```

Comment:
Has this key been documented in our internal Playbook?

Reply:
good point and done